### ISSUE TITLE
IPCW

### ISSUE THREAD

**Issue by ellessenne** [brittanyblouin/ANCRTAdjust#11]
Please include more details on the weighting procedure (e.g. mathematical formulae and so on) in the documentation.

Also, can the procedure handle intermittent missingness or is it assuming "censoring" in the sense that once unobserved, it will always be unobserved from there onwards?

**Comment by seabbs** [comment]
Agreed - this feels far too much like a black box to me. 

It would be nice to have a vignette on this - potentially with an example comparing using and not using (and/or some comment on other methods).

**Comment by m-maheu-giroux** [comment]
Fair point. The mathematical formula has now been included in the package documentation and it should now be clear how IPCW was implemented.

**Comment by ellessenne** [comment]
Thanks, but I still think you may want to include more details - how do you estimate the probability of censoring?

I would actually suggest adding a (short) vignette that describes the relevant methodology, I believe it is super important to not have any doubt regarding what the package does.
